coverage.py

```diff
@@ -72,14 +72,19 @@ def process_item(self, identifier):
         if isinstance(epub_path, CoverageFailure):
             return epub_path
 
-        license_pool = edition.license_pool
```

Comment:
This change made necessary by https://github.com/NYPL-Simplified/server_core/pull/629